Morphic/Morphic.tex

```diff
@@ -559,6 +559,31 @@ \subsection{Keyboard events}
 To discover the key values, you can open a Transcript window and add \glbind{Transcript} \ct{show: anEvent keyValue} to \mthref{handleKeystroke}.
 The \ct{anEvent} argument of \ct{handleKeystroke:} is an instance of \clsind{KeyboardEvent}, another subclass of \clsind{MorphicEvent}. Browse this class to learn more about keyboard events.
 
+\subsection{Global accessors}
```

Comment:
Maybe a more detailed heading could be "Global accessors for user interface objects"?

Reply:
Resolved via 54bf6db8700fd02c4d21d8f3af08e68843378390